pkg/api/controllers/profile.go

```diff
@@ -64,7 +64,7 @@ func (p *ProfilePortal) CreateProfile() {
 				profile.ProvisioningProperties.DataStorage.StorageAccessCapability = []string{"Read", "Write", "Execute"}
 			}
 		}
-		if io := pp.IOConnectivity; io.IsEmpty(){
+		if io := pp.IOConnectivity; io.IsEmpty() {
```

Comment:
just a line position change?! no code change..

Reply:
That's the impact after running goimports command.